### PR TITLE
KCCOV-17 Make GET route for resources

### PR DESCRIPTION
**Pull request by Misterimpulse**
JIRA: https://kccovid.atlassian.net/secure/RapidBoard.jspa?rapidView=1&projectKey=KCCOV&modal=detail&selectedIssue=KCCOV-17

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -28,3 +28,5 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+keys.json
```
Comment (Misterimpulse) [inline .gitignore:32]: If anyone knows a better way to store keys like this let me know! I'm a noob here lol

Comment (loganom) [reply .gitignore:32]: what's going to be in this file? keys for APIs we use or a DB?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,6 @@
 # site
 kccovid.com website
+
+# Building locally
+To build this locally, you will first have to add a "keys.json" file to your main directory containing the apiKey for
```
Comment (loganom) [inline README.md:5]: should have scanned the full PR - my questions have been answered

File: .gitignore (hunk 1 above)
Comment (loganom) [reply .gitignore:32]: we might consider moving these keys out into a service and calling that service for w/e data we need. Then we aren't exposing the keys (as they're used) to the world

File: src/server/server.js [hunk 3]
```diff
@@ -1,18 +1,77 @@
...
+    records.forEach((record) => {
+      const newField = buildResouce(record.fields);
+      airTableResposes.push(newField);
+    });
+
+    fetchNextPage();
+  };
+
+  // Callback to handle a finished call to airtable or a failure from the retrieval
+  const doneCallback = (error) => {
```
Comment (Zarex) [inline src/server/server.js:57]: do we want to cache the data in any way so that we don't need to make a call on every single request?

Comment (Misterimpulse) [reply src/server/server.js:57]: I like the idea but also I think we should make that a separate JIRA

**Pull request merged**